hooks.mako

```diff
@@ -1,7 +1,18 @@
 ## -*- coding: utf-8 -*-
 <%inherit file="base.mako" />
 
-<div class="page-header"><h1>Supported hooks</h1></div>
+<div class="page-header">
+    <h1>Supported hooks</h1>
+    <div class="filter">
+        <input id="search-hook-id" type="text" placeholder="filter hook id"></input>
+        <select id="select-hook-type">
```

Comment:
for some reason this select looks really broken on windows

would it be better to have labels for both of these inputs and then have the placeholder value here be `--` or something?  might make this a little nicer

Reply:
i think this should be fixed as i now use the bootstrap classes for styling the inputs.

added labels and moved the inputs out of the header, as the header would look really cluttered imo (and too high compared to the other sites due to the labels).